.github/workflows/ci.yml

```diff
@@ -23,6 +23,9 @@ env:
   INSTA_UPDATE: no
 
 jobs:
+  detect-long-path:
+    uses: ./.github/workflows/detect-long-path.yml
```

Comment:
could you put this in a separate PR, please. Let's get the snapshot names sorted first.

Reply:
Reverted

Reply:
Long path checker CI will be implemented in [another PR](https://github.com/matrix-org/matrix-rust-sdk/pull/4637).